src/cmdline.rs

```diff
@@ -409,6 +416,36 @@ pub struct IsoExtractPxeConfig {
     pub output_dir: String,
 }
 
+#[derive(Debug, StructOpt)]
+pub struct IsoExtractMinimalIsoConfig {
+    /// ISO image
+    #[structopt(value_name = "ISO")]
+    pub input: String,
+    /// Extract rootfs CPIO image as well
```

Comment:
We don't normally advertise that this is a CPIO archive; it's just the rootfs image.